x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -200,7 +207,15 @@ private FunctionDefinition[][] functions() {
                 def(MvMedian.class, MvMedian::new, "mv_median"),
                 def(MvMin.class, MvMin::new, "mv_min"),
                 def(MvSum.class, MvSum::new, "mv_sum"),
-                def(Split.class, Split::new, "split") } };
+                def(Split.class, Split::new, "split") },
+            // builtin functions
+            new FunctionDefinition[] {
```

Comment:
I've hacked this to make it show up in `SHOW FUNCTIONS`, but after discussing with @luigidellaquila it should live probably somewhere else.